recipes/gmp/all/patches/cpp23-unprototyped-functions.patch

```diff
@@ -0,0 +1,12 @@
+--- configure
```

Comment:
Reported in https://gmplib.org/list-archives/gmp-bugs/2025-February/005559.html by the lovely @dkm and fixed upstream in https://gmplib.org/list-archives/gmp-commit/2025-April/003259.html, accepting this as a valid backport which Compiler Explorer has been using for a few months now without issue https://github.com/compiler-explorer/compiler-explorer/issues/7360 :)